eng/common/templates/job/execute-sdl.yml

```diff
@@ -29,6 +29,12 @@ jobs:
   pool:
     name: Hosted VS2017
   steps:
+  - task: UseDotNet@2
```

Comment:
This should be done in arcade instead, this is part of `eng/common/*`.

Reply:
Yes they will be taking in the change as well, if the solution solves the problems